sources/packages/backend/libs/integrations/src/services/sfas/sfas-part-time-application.service.ts

```diff
@@ -44,6 +44,11 @@ export class SFASPartTimeApplicationsService
     application.CSGPAward = sfasApplication.CSGPAward;
     application.SBSDAward = sfasApplication.SBSDAward;
     application.extractedAt = getUTC(extractedDate);
+    application.CSPTAward = sfasApplication.CSPTAward;
```

Comment:
Please follow the camel case naming convention. 

If possible(optional) I would ask to refactor the existing ones as well.

Reply:
Thanks for the refactors.